### PR TITLE
feat(docs jobs): add docs pipeline jobs

### LINKED ISSUE #33: Build pipeline for deis.com + documentation
As we consolidate documentation into deis.com, we need to have a build pipeline for all of our docs/websites.

Upstream websites or docs projects (current):
- deis/workflow
- engineyard/deis.com

Any green build of master from should trigger a downstream deis/gutenberg job to deploy to staging. Successful deploy of staging should deploy to production.

Will have staging and prod IAM credentials for s3 upload and cloudfront distribution invalidation.

### PR DESCRIPTION
**Pull request by vdice**
TODO
- [x] amend the shell step in `deis_com.groovy` to run TBD container-ized `script/cibuild` (https://github.com/engineyard/deis.com/pull/152)
- [x] amend the shell step in `workflow.groovy` to run container-ized build/test via `make docker-build docker-test` (depends on https://github.com/deis/workflow/pull/300) 

Closes #33

### CODE REVIEW

File: jobs/deis_com.groovy [hunk 1]
```diff
@@ -0,0 +1,53 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = 'deis.com'
```
Comment (slack) [inline jobs/deis_com.groovy:3]: deis.com or deis_com, does it matter?

File: jobs/deploy_website.groovy [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  wrappers {
+    timestamps()
+    colorizeOutput 'xterm'
+    credentialsBinding {
+      file('GUTENBERG_STAGING_ENV', '7d29c809-9480-47ae-91aa-a25f43e58897')
+    }
+    parameters {
+      stringParam('WORKFLOW_DOCS_SOURCE', '${WORKSPACE}/workflow', 'Relative Workflow source')
+      stringParam('DEIS_COM_SOURCE', '${WORKSPACE}/deis.com', 'Relative Deis.com source')
+      stringParam('CONTAINER_ENV', '${GUTENBERG_STAGING_ENV}', 'Relative Workflow source')
```
Comment (slack) [inline jobs/deploy_website.groovy]: We should update this to be "Environment file with AWS API Keys, S3 Buckets and CloudFront values"... or something like that.

File: jobs/deis_com.groovy [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    }
+  }
+
+  steps {
+    shell '''
+      #!/usr/bin/env bash
+
+      set -eo pipefail
+
+      # containerized run of script/cibuild here
```
Comment (slack) [inline jobs/deis_com.groovy]: Should be able to call `make test` after https://github.com/engineyard/deis.com/pull/152 is merged!

File: jobs/deis_com.groovy (hunk 1 above)
Comment (vdice) [inline jobs/deis_com.groovy:3]: @slack I thought it did but tried the actual repo name of `deis.com` and no probs in Jenkins, so figured we'd go with that

Comment (slack) [inline jobs/deis_com.groovy:3]: Sounds good to me!

**Comment by sgoings** [comment]
Looks groovy to me...

**Comment by arschles** [comment]
LGTM3

**Pull request merged**